script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -5,190 +5,190 @@ My body temperature drops.
 The intense cold makes it difficult just to survive.//ruby dots <た|・><だ|・><生|・><き|・><る|・>
 My brain creaks as if it's being squeezed by my skull.
 <Page2>
-Predicting the outcome of the next two minutes, I want to tear out my heart, which is already ready to burst.
+Predicting the outcome of the next two minutes, I want to tear out my heart, which is already ready to burst.//Not sure where the 'I want to' is coming from here? 俺は２分後の結末を予測し、破裂しそうな心臓をかき毟る。
```

Comment:
I think its inaccurate yeah. Sentence might need to be reworded so that 'tear our my heart' doesn't sound like its literal

Reply:
Looking back on this I think it's probably fine.